.github/workflows/test-and-upload-coverage.yml

```diff
@@ -46,11 +46,13 @@ jobs:
             database-type: badger-memory
             mutation-type: collection-save
             detect-changes: false
-          - os: windows-latest
-            client-type: go
-            database-type: badger-memory
-            mutation-type: collection-save
-            detect-changes: false
+## TODO: https://github.com/sourcenetwork/defradb/issues/2449
```

Comment:
todo: This is the wrong issue, at least if you want this PR to resolve #2449 :)

Maybe link this to https://github.com/sourcenetwork/defradb/issues/2080 instead

Reply:
- [x] done